messaging/chat.py

```diff
@@ -35,6 +36,14 @@ def _record_message_in_database(self, sender, rapidpro_message_id, text, quick_r
                 # be confirmed with RapidPro
                 message.text = f'{message.text}{text}'
                 message.save(update_fields=['text'])
+            else:
+                payload = {
+                    'head': 'New Message!',
+                    'body': 'Click here to view it.',
+                    'url': self.thread.get_absolute_url(),
+                }
+                for user in self.thread.users.all():
+                    send_user_notification(user=user, payload=payload, ttl=1000)
```

Comment:
Is it possible for `send_user_notification` to fail? Does it throw exceptions that we may need to handle?

Reply:
yes, added try-except block and log the exception